spoon-runner/src/main/java/com/squareup/spoon/html/HtmlUtils.java

```diff
@@ -191,7 +192,9 @@ static ExceptionInfo processStackTrace(StackTrace exception) {
     if (exception == null) {
       return null;
     }
-    String message = exception.toString().replace("\n", "<br/>");
+    // Escape any special HTML characters in the exception that would otherwise break the HTML
+    // rendering (e.g. the angle brackets around the default toString() for enums).
+    String message = StringEscapeUtils.escapeHtml4(exception.toString());
```

Comment:
I think I'm fine with the \n replacement going away.  it's part of the original commit of that file and it doesn't have any unit tests, so it's hard to say whether it's still necessary.  @JakeWharton do you remember?